### ISSUE TITLE
Awaken icon

### ISSUE THREAD

**Issue by StWhiteCat** [kgirtxd/Genshin-Assets#1]
Hello, I am a CS beginner, thanks for your files. Could you please add some weapon icons (awaken)?

**Comment by kgirtxd** [comment]
I dont think this will be done anytime soon.  Might be better to query the weapon ID then pull from https://genshin.honeyhunterworld.com/
